### PR TITLE
Add basic documentation on Terraform Cloud

### PR DESCRIPTION
**Pull request by ferozsalam**

### CODE REVIEW

**Review by andreeleuterio — APPROVED** [review]
ty

File: content/departments/security/tooling/terraform-cloud.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+Sourcegraph uses Terraform Cloud to manage the deployment of cloud infrastructure on multiple
+platforms.
+
+In the majority of cases, Terraform Cloud should be transparent to developers: the main change to
+our default workflow is that for resources managed by Terraform Cloud, you will no longer be
+able to run `terraform apply` locally on your laptop. Instead:
+
+1. Open a pull request on a branch with your intended changes
+2. Terraform Cloud should run a `terraform plan` action and post the results as a GitHub status check.
+   You can also still run `terraform plan` locally as before (although the build will now occur in
```
Comment (evict) [inline content/departments/security/tooling/terraform-cloud.md]: Plan also happens in the cloud right? It's probably what you mean but the word 'build' confuses me here.

File: content/departments/security/tooling/terraform-cloud.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+in specified paths change`, and configure patterns to only trigger when the Terraform files you
+   care about are changed.
+2. In the same section, also select `Automatic speculative plans`. This enables the GitHub check
+   that occurs on opening a pull request.
+
+[migration]: https://developer.hashicorp.com/terraform/tutorials/cloud/cloud-migrate
+
+# Slack integration
+
+Terraform Cloud provides a Slack integration, which is also useful to stay notified of change that
```
Comment (evict) [inline content/departments/security/tooling/terraform-cloud.md]:
```suggestion
Terraform Cloud provides a Slack integration, which is also useful to stay notified of changes that
```

**Review by evict — APPROVED** [review]

File: content/departments/security/tooling/terraform-cloud.md (hunk 1 above)
Comment (ferozsalam) [reply content/departments/security/tooling/terraform-cloud.md]: Have clarified 👍

**Pull request merged**